core/src/packages/discord.ts

```diff
@@ -67,16 +73,17 @@ if (Token) {
 
 }
 
-async function SendMessage(urlEnd: string, body: any) {
-    const URL = `https://discord.com/api/v10/channels/${urlEnd}`
+async function SendMessage(urlEnd: string, body: any, type: string) {
```

Comment:
just use a string union for `type` - `'POST' | 'GET' | 'PATCH`. no need for `fetchTypes` (which should actually be called `fetchMethods`)